mips/intr.c

```diff
@@ -170,23 +170,24 @@ static void fpe_handler(exc_frame_t *frame) {
 }
 
 static void cp_unusable_handler(exc_frame_t *frame) {
-  int cp_id = (frame->cause & CR_CEMASK) >> CR_CESHIFT;
-  bool kernel_mode = in_kernel_mode(frame);
-
-  if (cp_id != 1) {
-    panic(
-      "Unexpected unusable coprocessor exception, with coprocessor id = %d\n",
-      cp_id);
+  if (in_kernel_mode(frame)) {
+    panic("FPU unusable exception in kernel mode.");
```

Comment:
This is not accurate. Other coprocessors may not be usable in kernel-mode as well, if they're present.

Let's `assert(cp_id < 2)` to make sure no-one tries to use other coprocessors.